example/flutter/pubspec.yaml

```diff
@@ -0,0 +1,6 @@
+name: usage_example
+dependencies:
+  usage:
+    path: ../..
+  flutter:
+    path: ../../../flutter/packages/flutter
```

Comment:
I'm not sure there are good options. The relative path will always bite people when they try and share flutter code. Env vars don't work here, but if they did, we could do something like `$FLUTTER_ROOT/packages/flutter`.